### PR TITLE
docs: add Onebeat to users list

### PR DESCRIPTION
**Pull request by GuyAttia**
Following the guide in `INTHEWILD.md`, this PR is adding [Onebeat](https://1beat.com/) as a Superset user

### CODE REVIEW

File: RESOURCES/INTHEWILD.md [hunk 1]
```diff
@@ -181,3 +181,4 @@ Join our growing community!
 - [Twitter](https://twitter.com/)
 - [VLMedia](https://www.vlmedia.com.tr/) [@ibotheperfect]
 - [Yahoo!](https://yahoo.com/)
+- [Onebeat](https://1beat.com/) [@GuyAttia]
```
Comment (villebro) [inline RESOURCES/INTHEWILD.md:184]: Please place the entry in the correct slot based on alphabetical ordering

**Review by villebro — CHANGES_REQUESTED** [review]
Thanks for the addition! One minor comment regarding ordering, other than that LGTM

File: RESOURCES/INTHEWILD.md (hunk 1 above)
Comment (GuyAttia) [reply RESOURCES/INTHEWILD.md]: Done

File: RESOURCES/INTHEWILD.md [hunk 2]
```diff
@@ -178,6 +178,7 @@ Join our growing community!
 - [Grassroot](https://www.grassrootinstitute.org/)
 - [komoot](https://www.komoot.com/) [@christophlingg]
 - [Let's Roam](https://www.letsroam.com/)
+- [Onebeat](https://1beat.com/) [@GuyAttia]
 - [Twitter](https://twitter.com/)
 - [VLMedia](https://www.vlmedia.com.tr/) [@ibotheperfect]
-- [Yahoo!](https://yahoo.com/)
+- [Yahoo!](https://yahoo.com/)
```
Comment (villebro) [inline RESOURCES/INTHEWILD.md:184]: It seems like you've removed the newline after the last entry - can you reintroduce it?

Comment (GuyAttia) [reply RESOURCES/INTHEWILD.md]: Done (:

**Review by villebro — APPROVED** [review]

**Pull request merged**